src/util/nft.js

```diff
@@ -194,19 +194,21 @@ function formatNFTEvent(event) {
     sender: fromWallet,
     receiver: toWallet,
     tx_hash: txHash,
+    action,
     timestamp,
     memo,
   } = event;
   let eventName;
-  switch (event.action) {
+  switch (action) {
     case '/cosmos.nft.v1beta1.MsgSend':
       eventName =
         fromWallet === LIKECOIN_NFT_API_WALLET ? 'purchase' : 'transfer';
       break;
+    case 'buy_nft':
```

Comment:
Should ask @joshkiu for new event icon?

Reply:
Confirmed with @joshkiu 
![image](https://user-images.githubusercontent.com/17264716/210489400-963a2ab2-74d9-4bbd-8910-b546f694b271.png)